device-types/Ubiquiti/UVC-G6-INS.yaml

```diff
@@ -0,0 +1,99 @@
+---
+manufacturer: Ubiquiti
+model: Unifi G6 Instant
+slug: ubiquiti-unifi-uvc-g6-ins
+part_number: UVC-G6-INS
+comments: |
```

Comment:
Thank you for your contribution,
Please reduce the amount of information in the comments property, this should only be a brief description with a link to the documentation.